amp-beta-tester.php

```diff
@@ -0,0 +1,292 @@
+<?php
+/**
+ * Plugin Name: AMP Beta Tester
+ * Description: Opt-in to receive non-stable release builds for the AMP plugin.
+ * Plugin URI: https://amp-wp.org
+ * Author: AMP Project Contributors
+ * Author URI: https://github.com/ampproject/amp-wp/graphs/contributors
+ * Version: 1.4.0-beta1
+ * Text Domain: amp
+ * Domain Path: /languages/
+ * License: GPLv2 or later
+ *
+ * @package AMP Beta Tester
+ */
+
+namespace AMP_Beta_Tester;
+
+define( 'AMP__BETA_TESTER__DIR__', dirname( __FILE__ ) );
+define( 'AMP__BETA__TESTER__RELEASES__TRANSIENT', 'amp_releases' );
+define( 'AMP__PLUGIN__BASENAME', 'amp/amp.php' );
+
+// DEV_CODE. This block of code is removed during the build process.
+if ( file_exists( AMP__BETA_TESTER__DIR__ . '/amp.php' ) ) {
+	add_filter(
+		'site_transient_update_plugins',
+		function ( $updates ) {
```

Comment:
No `$this` context, so closure can be static for performance reasons.
```suggestion
		static function ( $updates ) {
```

Reply:
Thanks, done in https://github.com/ampproject/amp-wp/pull/3560/commits/7c19a8e351362ea70e6c1c6ffbe0adfb097fbd67.